portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -68,7 +72,8 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       String comment = (String) entity.getProperty("message");
       String user = (String) entity.getProperty("name");
       String pictureURL = (String) entity.getProperty("picture");
-      comments.add(new Comment(user, comment, pictureURL));
+      String id = KeyFactory.keyToString(entity.getKey());
```

Comment:
optional: Could be cool to be able to create a comment from an Entity directly. You could add a static function to comment called createCommentFromEntity() that constructs and a new comment OR alternatively if you will only have create comments from entities you could just take in an entity in the constructor.